examples/feature_store/feast_feature_repo/example.py

```diff
@@ -0,0 +1,38 @@
+# This is an example feature definition file
+
+from feast import Entity, Feature, FeatureView, FileSource, ValueType
+from google.protobuf.duration_pb2 import Duration
+
+# Read data from parquet files. Parquet is convenient for local development mode. For
+# production, you can use your favorite DWH, such as BigQuery. See Feast documentation
+# for more info.
+driver_hourly_stats = FileSource(
+    path="/Users/strickvl/coding/zenml/repos/zenml/examples/feature_store/feast_feature_repo/data/driver_stats.parquet",
```

Comment:
Local path :-) Better to use relative paths here

Reply:
oops this is auto-generated. I'll fix.